CRM/Activity/Form/Activity.php

```diff
@@ -727,8 +727,7 @@ public function buildQuickForm() {
       $this->getElement('source_contact_id')->freeze();
     }
 
-    //need to assign custom data type and subtype to the template
-    $this->assign('customDataType', 'Activity');
```

Comment:
Is this line removed intentionally @eileenmcnaughton ? I'm not able to load custom field while adding a case activity due to this removal - `civicrm/case/activity?action=add&reset=1&cid=2&caseid=2&atype=14`

Reply:
maybe `templates/CRM/Case/Form/Activity.tpl` requires the same treatment here https://github.com/civicrm/civicrm-core/blob/master/templates/CRM/Case/Form/Activity.tpl#L142, since it calls the same PHP file.

Reply:
thanks @jitendrapurohit - I added it in - I also added the entity type on the Case file for case for safety when we get to that one

Reply:
Awesome, confirmed the activity custom data load fine on case activity now. Thanks.